satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -424,6 +424,20 @@ model bucket_storage_tally (
 	field  metadata_size      uint64
 )
 
+create bucket_storage_tally ( )
+
+// for rolling up a period
+read all (
+	select bucket_storage_tally
+	where bucket_storage_tally.interval_start >= ?
+	where bucket_storage_tally.interval_start < ?
+)
+
+// for deleting after rollup
+delete bucket_storage_tally (
+	where bucket_storage_tally.interval_start >= ?
+	where bucket_storage_tally.interval_start < ?
+)
```

Comment:
are you using these `read all` and `delete bucket_storage_tally`? If not, I feel like you can remove them here.